packages/cli/src/execution-lifecycle/execution-lifecycle-hooks.ts

```diff
@@ -166,29 +220,23 @@ function hookFunctionsPreExecute(): IWorkflowExecuteHooks {
 	};
 }
 
+function hookFunctionsFinalizeExecutionStatus(): IWorkflowExecuteHooks {
```

Comment:
We might want a docline to warn that this should be placed always before the `*Save` hook bundles. Or move it there to the top if possible.